hack/test.sh

```diff
@@ -4,12 +4,8 @@
 export GO111MODULE=on
 export GOFLAGS=-mod=vendor
 
-# Test if we can build the program
-echo "Building virtual cluster..."
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build cmd/vcluster/main.go || exit 1
```

Comment:
Why was this removed? :) 
I think it catches compilation errors nice and early and provides simpler output - easier to see compilation errors.

Reply:
And I think it wouldn't slow down the testing much because go test would reuse the build cache populated by this go build command.